src/mjolnir/graphbuilder.cc

```diff
@@ -1149,33 +1095,35 @@ void BuildLocalTiles(const unsigned int thread_count,
 namespace valhalla {
 namespace mjolnir {
 
-// Build the graph from the input
-void GraphBuilder::Build(const boost::property_tree::ptree& pt,
-                         const OSMData& osmdata,
-                         const std::string& ways_file,
-                         const std::string& way_nodes_file,
-                         const std::string& nodes_file,
-                         const std::string& edges_file,
-                         const std::string& complex_from_restriction_file,
-                         const std::string& complex_to_restriction_file) {
-  std::string tile_dir = pt.get<std::string>("mjolnir.tile_dir");
-  unsigned int threads =
-      std::max(static_cast<unsigned int>(1),
-               pt.get<unsigned int>("mjolnir.concurrency", std::thread::hardware_concurrency()));
+// Line up the nodes and then re-map the edges that the edges to them
```

Comment:
the end of this comment might have a typo in it

Reply:
Yup fixing!